### PR TITLE
Use variable install path instead of fixed install path.

### PR DESCRIPTION
**Pull request by hillwoodroc**

### CODE REVIEW

**Comment by zccrs** [comment]
Thank you. But you can change the installation path of the files by specifying the value of INSTALL_ROOT when make

File: deepin-clone.pro [hunk 1]
```diff
@@ -1,3 +1,4 @@
+PREFIX = /usr
```
Comment (rafaeldev) [inline deepin-clone.pro]: 👍

Comment (zccrs) [inline deepin-clone.pro]: the "PREFIX" d'not defined in this, Doing so will not make it valid in other pro files.
Can move to the "common.pri" file
````
isEmpty(PREFIX): PREFIX = /usr
````

**Comment by justforlxz** [comment]
Sorry, this issue will be closed soon. If it is necessary to discuss it again, please create a new issue.

**Pull request closed without merging**